src/components/@shared/FormInput/InputElement/Credential/index.tsx

```diff
@@ -37,6 +39,14 @@ export default function Credentials(props: InputProps) {
     setValue('')
   }
 
+  const showAddressName = (address: string) => {
+    const addressKey = Object.keys(verifiedAddresses).filter((key) => {
+      return key.toLowerCase() === address.toLowerCase()
+    })
```

Comment:
suggestion(blocking): instead of filtering, you could use the `find` function to only return the first hit for any given address